pkg/driver/camera/camera.go

```diff
@@ -1,3 +1,13 @@
+/*
+Package camera provides a video camera driver.
+
+Device Label Generation Rules:
```

Comment:
```suggestion
Device Label Generation Rules

```
to make it header.

current:
![image](https://user-images.githubusercontent.com/8390204/109824068-d3226380-7c7b-11eb-94f1-b82391b65add.png)

without trailing `:` and with newline:
![image](https://user-images.githubusercontent.com/8390204/109824157-e7fef700-7c7b-11eb-94ba-da31216ba1d1.png)

Reply:
Duh!